mount.go

```diff
@@ -99,6 +100,25 @@ be interacting with the file system.`)
 	}
 	bm := gcsx.NewBucketManager(bucketCfg, storageHandle)
```

Comment:
We are passing bucketCfg value to create BucketManager. And then modifying the value. This doesn't seem correct.

Reply:
yeah, this looks like a bug. Moved it before NewBucketManager .

Reply:
Do we have test plan to validate these kind of regression?